x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/evaluate_panel.tsx

```diff
@@ -349,14 +374,25 @@ export const EvaluatePanel: FC<Props> = ({ jobConfig, jobStatus, searchQuery })
                           <EuiDataGrid
                             data-test-subj="mlDFAnalyticsClassificationExplorationConfusionMatrix"
                             aria-label="Classification confusion matrix"
-                            columns={columns}
+                            columns={shownColumns}
                             columnVisibility={{ visibleColumns, setVisibleColumns }}
-                            rowCount={columnsData.length}
+                            rowCount={rowCount}
                             renderCellValue={renderCellValue}
                             inMemory={{ level: 'sorting' }}
-                            toolbarVisibility={false}
+                            toolbarVisibility={{
+                              showColumnSelector: true,
+                              showStyleSelector: true,
```

Comment:
I'm not convinced the 'Density' control is worth showing. Even when the table has more than 10 columns, the default setting seems to be good enough, particularly as you can always resize columns. Happy to be outvoted though ;)

Reply:
I agree we could hide it. For the tables in the transform wizard we also hide it.

Reply:
Hidden in 422c7aacdfc9d129a7877af2d9f1c8ebede15f80